packages/expo-updates/src/Updates.types.ts

```diff
@@ -97,10 +97,23 @@ export type UpdateFetchResultFailure = {
   manifest: undefined;
 };
 
+/**
+ * The rollback to embedded result of fetching a new update.
+ */
+type UpdateFetchResultRollbackToEmbedded = {
+  /**
+   * Signifies that the update was a roll back to the embedded update.
+   */
+  isRollBackToEmbedded: true;
```

Comment:
```suggestion
  isRollBackToEmbedded: true;
  /**
   * This is false since we are rolling back to the embedded update.
   */
  isNew: false;
  /**
   * No manifest, since this is a roll back to the embedded update.
   */
  manifest: undefined;
```

Reply:
Also wondering if `isRollBackToEmbedded: false;` should be added to the other two `UpdateFetchResult` types.

Reply:
The way the native code is written, we don't include those properties when `isRollBackToEmbedded` is true. We can re-assess in a follow-up if necessary.